tests/common/fixtures/duthost_utils.py

```diff
@@ -498,12 +498,12 @@ def _dut_qos_map(dut):
 
 
 @pytest.fixture(scope='class')
-def dut_qos_maps(get_src_dst_asic_and_duts):
+def dut_qos_maps(rand_selected_front_end_dut):
```

Comment:
I remember this fixture is also used by other test script. Can you please double check?
If it's also used by other test script, the change can probably cause regression.